app/sidekiq/lighthouse/create_intent_to_file_job.rb

```diff
@@ -51,14 +55,20 @@ class CreateIntentToFileError < StandardError; end
     # @param [ActiveSupport::TimeWithZone] form_start_date
     # @param [String] veteran's ICN
     #
-    def perform(form_type, form_start_date, veteran_icn)
-      init(form_type, veteran_icn)
+    def perform(in_progress_form_id, veteran_icn, participant_id)
+      init(in_progress_form_id, veteran_icn)
 
-      @itf_log_monitor.track_create_itf_begun(@itf_type, form_start_date, @user_account&.id)
+      @itf_log_monitor.track_create_itf_begun(@itf_type, @form_start_date, @user_account&.id)
       @service.create_intent_to_file(@itf_type, '')
-      @itf_log_monitor.track_create_itf_success(@itf_type, form_start_date, @user_account&.id)
+      @itf_log_monitor.track_create_itf_success(@itf_type, @form_start_date, @user_account&.id)
```

Comment:
could have the monitor and service in "null setter" functions `||=` and the other instance variables with `attr_reader` to have this a little cleaner